nmtwizard/preprocess/preprocess.py

```diff
@@ -111,12 +111,11 @@ def _generate_models(self, tokenization_step, option):
         opt_source = tok_config.get('source', {}).get(build_option)
         opt_target = tok_config.get('target', {}).get(build_option)
 
-        if not opt_multi and not (opt_source and opt_target):
+        if not opt_multi and not opt_source and not opt_target:
```

Comment:
Not sure why this change. For subword / vocabulary building we want the options to be specified either in multi or in both source and target, don't we ?

Reply:
I think we also need to allow building for just the source, or just the target.

Reply:
Ok, it's just a detail.